net-p2p/ogmios/Makefile

```diff
@@ -0,0 +1,72 @@
+PORTNAME=       ogmios
+PORTVERSION=    5.5.0
+DISTVERSIONPREFIX=v
+CATEGORIES=     net-p2p
+EXTRACT_ONLY+=  input-output-hk-libsodium-${LIBSODIUM_HASH}_GH0${EXTRACT_SUFX} \
+                bitcoin-core-secp256k1-${SECP256K1_HASH}_GH0${EXTRACT_SUFX}
+
+MAINTAINER=     arrowd@FreeBSD.org
+COMMENT=        Offers a WebSockets API that enables local clients to speak Ouroboros' mini-protocols via JSON/RPC.
+
+LICENSE=        APACHE20
+
+BUILD_DEPENDS=  bash:shells/bash \
+                git:devel/git
+
+USES=           autoreconf:build cabal gmake libtool pkgconfig
+
+USE_GITHUB=     yes
+GH_ACCOUNT=     cardanosolutions bitcoin-core:secp input-output-hk:sodium
+GH_PROJECT=     ${PORTNAME} libsodium:sodium secp256k1:secp
+GH_TAGNAME=     ${LIBSODIUM_HASH}:sodium ${SECP256K1_HASH}:secp
+
+USE_RC_SUBR=    ogmios
+
+LIBSODIUM_HASH=         66f017f16633f2060db25e17c170c2afa0f2a8a1
+# Obtained from https://github.com/input-output-hk/cardano-node/blob/master/.github/workflows/github-page.yml#L21
+SECP256K1_HASH=         ac83be33d0956faf6b7f61a60ab524ef7d6a473a
+LIBS_PREFIX=            ${WRKDIR}/libs_install
+
+WRKSRC_SUBDIR= server
+
+CABAL_PROJECT=          append
+EXECUTABLES=            exe:ogmios
+SKIP_CABAL_PLIST=       yes
```

Comment:
Will do... but maybe we should keep the pkg-plist file itself? since i found from porters handbook: 

"Usage of PLIST_FILES should not be abused. When looking for the origin of a file, people usually try to grep through the pkg-plist files in the ports tree. Listing files in PLIST_FILES in the Makefile makes that search more difficult.

Reply:
Changed back to SKIP_CABAL_PLIST=       yes   so that it build again, unless further instructions provided.